src/content/app/genome-browser/components/track-settings-panel/TrackSettingsPanel.scss

```diff
@@ -22,12 +22,17 @@
       line-height: 30px;
       font-size: 12px;
       font-weight: 300;
-      padding-right: 4px;
+      padding-right: 10px;
     }
 
     &:not(:last-child) {
       border-right: 1px solid $grey;
     }
+
+    &:last-child {
+      padding: 0 25px;
+      min-height: 62px;
```

Comment:
1) what's the purpose of the `min-height`?
2) should the padding be the property of the `moveTracksIcon` class?